app/src/main/java/org/jellyfin/androidtv/JellyfinApplication.kt

```diff
@@ -1,10 +1,7 @@
 package org.jellyfin.androidtv
 
 import android.content.Context
-import androidx.lifecycle.Lifecycle
-import androidx.lifecycle.LifecycleObserver
-import androidx.lifecycle.OnLifecycleEvent
-import androidx.lifecycle.ProcessLifecycleOwner
+import androidx.lifecycle.*
```

Comment:
Think this will conflict with the other PR?

Reply:
Yup!